### ISSUE TITLE
Support `planet` command on arm64

### ISSUE THREAD

**Issue by sky1045** [planetarium/libplanet#2161]
```
$ dotnet --version
6.0.302

$ dotnet tool install libplanet.tools -g
$ planet
You must install .NET to run this application.

App: /Users/<username>/.dotnet/tools/planet
Architecture: x64
App host version: 6.0.7
.NET location: Not found

Learn about runtime installation:
https://aka.ms/dotnet/app-launch-failed

Download the .NET runtime:
https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=osx.12-x64&apphost_version=6.0.7
```

libplanet.tools seems to support only x64 arch, so it cannot find .NET runtime on arm64

**Comment by dahlia** [comment]
I guess it will be resolved when we upgrade .NET SDK to 6.0.  (Currently we're primarily using .NET Core SDK 3.1.)

**Comment by SieR-VR** [comment]
✋

**Comment by longfin** [comment]
I've tried to change `<TargetFramework>` value on [here](https://github.com/planetarium/libplanet/blob/0.39.0/Libplanet.Tools/Libplanet.Tools.csproj#L23).to `.net6.0`, but on my system, it failed as below

```
[longfin@longfin-desktop-arch Libplanet.Tools]$ dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.1.120
 Commit:    6f5755c372

Runtime Environment:
 OS Name:     arch
 OS Version:  
 OS Platform: Linux
 RID:         arch-x64
 Base Path:   /usr/share/dotnet/sdk/3.1.120/

Host (useful for support):
  Version: 6.0.2
  Commit:  839cdfb0ec

.NET SDKs installed:
  3.1.120 [/usr/share/dotnet/sdk]
  6.0.102 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.20 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.2 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.20 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.2 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
  
[longfin@longfin-desktop-arch Libplanet.Tools]$ dotnet build
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 13.45 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Stun/Libplanet.Stun.csproj.
  Restore completed in 13.44 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Tools/Libplanet.Tools.csproj.
  Restore completed in 13.44 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.RocksDBStore/Libplanet.RocksDBStore.csproj.
  Restore completed in 13.54 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Net/Libplanet.Net.csproj.
  Restore completed in 13.44 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Extensions.Cocona/Libplanet.Extensions.Cocona.csproj.
  Restore completed in 13.51 ms for /home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet/Libplanet.csproj.
/usr/share/dotnet/sdk/3.1.120/Microsoft.Common.CurrentVersion.targets(1175,5): error MSB3644: The reference assemblies for .NETFramework,Version=v6.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Tools/Libplanet.Tools.csproj]

Build FAILED.

/usr/share/dotnet/sdk/3.1.120/Microsoft.Common.CurrentVersion.targets(1175,5): error MSB3644: The reference assemblies for .NETFramework,Version=v6.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/home/longfin/9c-launcher/NineChronicles.Headless/Lib9c/.Libplanet/Libplanet.Tools/Libplanet.Tools.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.37
```

After digging about it, I found that `global.json` is forcing the sdk to `netcore3.1`. when I remove whole content of `global.json`, it worked as expected. 

@dahlia Are there any required steps before bumping sdk version? cc @tkiapril

**Comment by tkiapril** [comment]
@longfin afaik bumping global.json to 6.0 should suffice, it's included in #2173.
